configuration/sitemaps.md

```diff
@@ -555,31 +546,34 @@ If an operator is not specified, the operator will default to `==`.
 
 The comparison operators for `labelcolor` and `valuecolor` are the same as for the visibility parameter.
 
-Examples:
+**Examples:**
+
+The following three lines are equivalent.
 
 ```perl
-Text item=Weather valuecolor=[Temperature<=4="blue"]
-                  labelcolor=[Temperature<=4="blue"]
-Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
-                                  >=25="orange", >=15="green", 0="white", <15="blue"]
+Text item=Temperature labelcolor=[>0="blue"] valuecolor=[22="green"]
+Text item=Temperature labelcolor=[>0="blue"] valuecolor=[==22="green"]
+Text item=Temperature labelcolor=[>0="blue"] valuecolor=[Temperature==22="green"]
 ```
 
-In the first example, the `Weather` Item is blue (label and value) if the `Temperature` is at or below 4°C.
+The line below illustrates the importance of operator order:
 
-The order in which expressions are listed is important.
-Note that expressions are evaluated from left to right, and the first matching expression determines the color.
-Looking at the second example, you will see that the order of the expressions is set so that the first match, left-to-right gives the correct result.
-If the order was reversed, color assignment would not work properly.
+```perl
+Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
+                                  >=25="orange", >=15="green", 0="white", <15="blue"]
+```
 
-In the second example, the effect of omitting the `item_name` and `operator` parameters is shown by omitting `Temperature` and by omitting the comparison operator in the expression `0="white"` (as compared to `==0="white"`).
+Note that expressions are evaluated from left to right; the first matching expression determines the color.
+If the order of the expressions was reversed, the color assignment would not work properly.
+Note also, the effect of omitting `Temperature` and the comparison operator in the expression `0="white"` (as compared to `==0="white"`).
 
-The following three lines are equivalent, and all of them are valid:
+The two lines below show that `item_name` may be omitted, in which case, the name from the previous line will be used:
 
 ```perl
-Text item=Temperature labelcolor=[>0="blue"] valuecolor=[22="green"]
-Text item=Temperature labelcolor=[>0="blue"] valuecolor=[==22="green"]
-Text item=Temperature labelcolor=[>0="blue"] valuecolor=[Temperature==22="green"]
+Text item=Weather valuecolor=[Temperature<=4="blue"]
+                  labelcolor=[Temperature<=4="blue"]
 ```
+The `Weather` Item will be blue (label and value) if `Temperature` is less than or equal to 4°C.
 
 ![Presentation of the color parameters in BasicUI](images/sitemap_demo_colors.png)
```

Comment:
Does the example still make sense? Seems like it doesn't fit.

Reply:
Agree.  See below.